composer/algorithms/alibi/gpt2_alibi.py

```diff
@@ -42,7 +42,7 @@ def _attn(self, query, key, value, attention_mask=None, head_mask=None):
 def enlarge_mask(module: torch.nn.Module, max_sequence_length: int):
     old_mask = module.bias
     new_mask = torch.tril(
-        torch.ones((max_sequence_length, max_sequence_length), dtype=torch.uint8,
+        torch.ones((max_sequence_length, max_sequence_length), dtype=torch.uint8,  # type: ignore
```

Comment:
Why are new `type: ignore` statements needed? Are these just to silence warnings?

Reply:
It might be a pyright update, not sure. I was getting some errors.

Reply:
![image](https://user-images.githubusercontent.com/87037432/141835890-0f6483c4-58a4-41d2-a178-14fd79084a1f.png)